main/src/main/java/com/ldtteam/blockout/connector/client/ClientSideOnlyGuiController.java

```diff
@@ -72,39 +70,39 @@ public void openUI(@Nullable final UUID playerId, @NotNull final IGuiKey key)
 
             if (host.getAllCombinedChildElements().entrySet().stream().anyMatch(e -> e instanceof Slot))
             {
-                throw new IllegalArgumentException("Can not open UI that holds Slots. Inventories are not supported in ClientSide only gui's.");
+                throw new IllegalArgumentException("Can not open UI that holds Slots. Inventories are not supported in ClientSide only guitemp's.");
             }
         }
         catch (IllegalArgumentException ex)
         {
-            Log.getLogger().error("Failed to create client side only gui.", ex);
+            Log.getLogger().error("Failed to build client side only guitemp.", ex);
```

Comment:
Another intellij text refactor? (there are a few other places, without my comments)

Reply:
Yep. Will try to find them.